src/lms-ds-loader/README.md

```diff
@@ -5,22 +5,26 @@ Model (LMS-UDM) into a Learning Management System Data Store (LMS-DS) database.
 
 Limitations as of September 23, 2020:
 
-* Only supports SQL Server (tested on MSSQL 2019)
-* Only supports loading User files
+* Data loads only supports SQL Server (tested on MSSQL 2019).
+* Only supports loading User files.
 * Does not perform updates or deletes, and will throw an error if trying to
   reload an existing record.
 
 ## Getting Started
 
+1. SQL Server support requires the Microsoft ODBC 17 driver, which is newer than
+   the ones that come with most operating systems.
+   * Windows: `choco install sqlserver-odbcdriver`
+   * [Linux
+     instructions](https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-ver15)
+     (has not been tested yet)
 1. Requires Python 3.8+ and Poetry.
 1. Install required Python packages:
 
    ```bash
    poetry install
    ```
 
-1. Run the two SQL scripts in `src/lms-ds-installer/mssql` in a SQL Server
-   database.
 1. The database account used when running the tool needs to be a member of the
    following roles in the destination database:
 

```

Comment:
the sample run of line 43 needs to be updated

Reply:
Updated it, and made several other readme improvements in the latest commit. I've also added "readme improvement" acceptance criteria to several other tickets so that we can continuously improve on this.